README.md

```diff
@@ -27,44 +99,66 @@ It correctly bundles React in production mode and optimizes the build for the be
 The build is minified and the filenames include the hashes.\
 Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+To run test coverages, run the following command:
+
+```sh
+  npm test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="author"></a>
+
+>
+
+👤 **Abu Talha**
+
+- GitHub: [@githubhandle](https://github.com/AbuTalha3)
+- Twitter: [@twitterhandle](https://twitter.com/AbuTalha8T)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abu-talha-8203b252/)
+
```

Comment:
- [ ] Kindly fix the broken link to the `LinkedIn` profile for a more professional readme file.
![image](https://github.com/AbuTalha3/my-react-app/assets/81932478/a8e92586-2e6b-45b7-8d90-6d596f39cfc1)